docs/architecture_overview/master03-aims.adoc

```diff
@@ -108,7 +108,7 @@ Ultimately any software and information architecture only provides utility when
 .Community Shared-care Context
 image::{diagrams_uri}/shared_ehr.svg[id=shared_ehr, align="center", width=70%]
 
-In this form, the openEHR services are added to the existing IT infrastructure to provide a shared, secure health record for patients that are seen by any number of health providers in their community context. openEHR-enabled systems can also be used to provide EMR/EPR functionality at provider locations. Overall, a number of important categories of system can be implemented using openEHR including the following:
+In this form, the openEHR services are added to the existing IT infrastructure to provide a shared, secure health record for patients that are seen by any number of health providers in their community context. openEHR-enabled systems can also be used to provide EMR/EPR functionality at provider locations. Overall, a number of important categories of systems can be implemented using openEHR including the following:
```

Comment:
I guess since a category consists always of multiple elements of things, the thing it refers to needs to be in plural form.

Reply:
The original is actually correct grammar, it's like saying 'types of car' (you don't say 'types of cars') etc. Admittedly, English is not very systematic on this score ;)